docs/shared/console.md

```diff
@@ -0,0 +1,50 @@
+# Nx Console
+
+<figure>
+  <img
+    src="/assets/content/shared/nx-console-logo.png"
+    alt="Nx Console logo"
+    style="max-width: 100%;"
+  />
+</figure>
+
+**Spend less time looking up command line arguments and more time shipping incredible products.**
+
+- [Install from the VSCode Marketplace](https://marketplace.visualstudio.com/items?itemName=nrwl.angular-console)
+- [Contribute on GitHub](https://github.com/nrwl/nx-console)
+
+<figure>
+  <img
+    src="/assets/content/shared/nx-console-screenshot.png"
+    alt="Nx Console screenshot"
+    style="max-width: 100%;"
+  />
+</figure>
+
+With Nx, you can get a full-stack application up and running in minutes, no need to figure out source maps, webpack, test runners. It all works out of the box. Nx also helps you enforce consistent development practices by generating components, services, and state management modules.
+
+## Why Nx Console?
+
+Professional developers use both command-line tools and user interfaces. They commit in the terminal, but resolve conflicts in VSCode or WebStorm. They use the right tool for the job.
+
+Nx is a command-line tool, which works great when you want to serve an application or generate a simple component. But it falls short once you start doing advanced things.
+
+For instance:
+
+- Exploring custom schematic collections is hard in the terminal, but it's easy using Nx Console.
+- Using rarely-used flags is challenging. Do you pass absolute or relative paths? You don't have to remember any flags, names, or paths -- Nx Console will help you by providing autocompletion and validating your inputs.
+- Finding the right Nx extension can take a long time. When using Nx Console, you can find and install an extension in minutes.
+
+Nx Console does all that and more!
+
+## Download
+
+For VSCode users, you can install the [Nx Console VSCode Plugin](https://marketplace.visualstudio.com/items?itemName=nrwl.angular-console) from Marketplace.
+
+## True UI for Nx
+
+Nx Console is an UI for Nx. It will work for any schematic or any architect commands. Nx Console does not have a specific UI for, say, generating a component. Instead, Nx Console does what the command-line version of Nx does--it analyzes the same meta information to create the needed UI. This means that anything you can do with Nx, you can do with Nx Console. After all, Nx Console is the UI for Nx.
```

Comment:
```suggestion
Nx Console is the UI for Nx. It will work for any schematic or any architect commands. Nx Console does not have a specific UI for, say, generating a component. Instead, Nx Console does what the command-line version of Nx does--it analyzes the same meta information to create the needed UI. This means that anything you can do with Nx, you can do with Nx Console. After all, Nx Console is the UI for Nx.
```